devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -203,7 +181,7 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
-    if (cmd.value == 0 && device.latestState("color").value != "#ffffff") {
+    if (device.latestState("color") == null || (cmd.value == 0 && device.latestState("color").value != "#ffffff")) {
         sendEvent(name: "color", value: "#ffffff", displayed: true)
```

Comment:
It appears that a comment I added in the code did not appear. I am going to update that. Basically, the device changes the color to #ffffff in its firmware after off.